adal/util.py

```diff
@@ -55,7 +55,11 @@ def add_default_request_headers(self, options):
     headers[AdalIdParameters.SKU] = AdalIdParameters.PYTHON_SKU
     headers[AdalIdParameters.VERSION] = adal.__version__
     headers[AdalIdParameters.OS] = sys.platform
-    headers[AdalIdParameters.CPU] = 'x64' if platform.architecture()[0] == '64bit' else 'x86'
+
+    try:
+        headers[AdalIdParameters.CPU] = 'x64' if platform.architecture()[0] == '64bit' else 'x86'
+    except:
+        headers[AdalIdParameters.CPU] = 'unknown'

```

Comment:
This may not return the correct architecture in all environments:
http://stackoverflow.com/a/24944155

I had initially thought of it, but better return none than an incorrect value. How does the service use the header? For logging purposes?